web/modules/custom/jcc_custom/jcc_custom.install

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * @file
+ * Install taxonoomy terms for case served and service category.
+ */
+
+use Drupal\taxonomy\Entity\Term;
+use Drupal\taxonomy\Entity\Vocabulary;
+
+/**
+ * Implements hook_update_N().
+ */
+function jcc_custom_update_8064() {
+  // Vocabulary data.
+  $vocabularies = [
+    'cases_served' => 'All Cases Served',
+    'service_categories' => 'Service Categories',
+  ];
+
+  foreach ($vocabularies as $vid => $vocabulary_name) {
+    // Check if the vocabulary already exists.
+    $vocabulary = Vocabulary::load($vid);
+    if (!$vocabulary) {
+      // Create the vocabulary if it doesn't exist.
+      $vocabulary = Vocabulary::create([
```

Comment:
Is this still needed?  Verify vocabularies are created by configs in immutable; not dbup.